start_cerberus.py

```diff
@@ -119,6 +126,16 @@ def main(cfg):
                 for namespace, failures in failed_pods_components.items():
                     logging.info("%s: %s", namespace, failures)
 
+            for namespace in failed_pods_components.keys():
+                dir_name = "reports/" + namespace + '-logs'
+                if os.path.isdir(dir_name):
+                    runcommand.invoke("rm -R " + dir_name)
```

Comment:
The data collection needs to be optional, the user should be able to turn the collection off and on in the config.